### ISSUE TITLE
Add Font Awesome CSS to the Jekyll template

### ISSUE THREAD

**Issue by prycrane** [canada-ca/blogue-canada-ca-blog#112]
Please add the link to the font awesome CSS to the Jekyll template.

https://use.fontawesome.com/releases/v5.15.4/css/all.css

**Comment by delisma** [comment]
Would you want it as a default on all the pages or as a property on page-to-page basis?

**Comment by prycrane** [comment]
That's a good question.  I don't think there is much harm in making it available, but if it's rarely used, then it's a waste.  Why don't we put it at the page level, then people can use it with intension.

**Comment by delisma** [comment]
Agree. And I'm thinking of a feature that I could push to GCWeb-Jekyll in the future where you could have a property for Font Awesome (FA) itself and you could choose which version of FA to use on your page. That could facilitate the migration to newer version of FA in the long run.
